testplan/runnable/base.py

```diff
@@ -214,7 +214,7 @@ def success(self):
 
 
 class TestRunner(Runnable):
-    r"""
+    """
```

Comment:
Why have you made this change? Please revert - the `r"""` prefix is required so that the literal `\`s in the docstring aren't treated as escape sequences, which trigger warnings on Python 3.6+.